examples/step-1/step-1.cc

```diff
@@ -99,31 +100,40 @@ void second_grid ()
+  // straigth lines, and all cells are bi-linear quads, and that they
+  // are defined by the cells of the coarse grid (which we just
+  // created). It uses this information when cells are refined and new
+  // points need to be introduced; if the domain is assumed to be
+  // flat, then new points will simply be in the middle of the
   // surrounding ones.
   //
-  // Here, however, we would like to have a curved boundary. Fortunately, some
-  // good soul implemented an object which describes the boundary of a ring
-  // domain; it only needs the center of the ring and automatically figures
-  // out the inner and outer radius when needed. Note that we associate this
-  // boundary object with that part of the boundary that has the "boundary
-  // indicator" zero. By default (at least in 2d and 3d, the 1d case is
-  // slightly different), all boundary parts have this number, but you can
-  // change this number for some parts of the boundary. In that case, the
-  // curved boundary thus associated with number zero will not apply on those
-  // parts with a non-zero boundary indicator, but other boundary description
-  // objects can be associated with those non-zero indicators. If no boundary
-  // description is associated with a particular boundary indicator, a
-  // straight boundary is implied. (Boundary indicators are a slightly
-  // complicated topic; if you're confused about what exactly is happening
-  // here, you may want to look at the
-  // @ref GlossBoundaryIndicator "glossary entry on this topic".)
-  const HyperShellBoundary<2> boundary_description(center);
-  triangulation.set_boundary (0, boundary_description);
+  // Here, however, we know that the domain is curved, and we would
+  // like to have the Triangulation place new points according to the
+  // underlying geometry. Fortunately, some good soul implemented an
+  // object which describes a spherical domain, of which the ring is a
+  // section; it only needs the center of the ring and automatically
+  // figures out how to instruct the Triangulation where to place the
+  // new points. We first set the "manifold indicator" of all cells
+  // and faces of the Triangulation to the value zero, and then
+  // associate the curved Manifold object with those parts of the
+  // Triangulation that have the "manifold indicator" zero. By
+  // default, all cells and faces of the Triangulation have their
+  // manifold_id set to numbers::invalid_manifold_id, which is the
+  // default if you want a flat manifold, but you can change this
+  // number for individual cells and faces. In that case, the curved
+  // manifold thus associated with number zero will not apply on those
+  // parts with a non-zero manifold indicator, but other manifold
+  // description objects can be associated with those non-zero
+  // indicators. If no manifold description is associated with a
+  // particular manifold indicator, a flat manifold is
```

Comment:
again, maybe replace "flat manifold" by "a manifold that produces straight edges"?